DataFusion/src/V1/Instance_DisabledReason.php

```diff
@@ -0,0 +1,16 @@
+<?php
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
+# source: google/cloud/datafusion/v1/datafusion.proto
+
+namespace Google\Cloud\DataFusion\V1;
+
+if (false) {
+    /**
+     * This class is deprecated. Use Google\Cloud\DataFusion\V1\Instance\DisabledReason instead.
+     * @deprecated
+     */
+    class Instance_DisabledReason {}
+}
+class_exists(Instance\DisabledReason::class);
+@trigger_error('Google\Cloud\DataFusion\V1\Instance_DisabledReason is deprecated and will be removed in a future release. Use Google\Cloud\DataFusion\V1\Instance\DisabledReason instead', E_USER_DEPRECATED);
```

Comment:
This class should not be here. We need to merge https://github.com/googleapis/google-cloud-php/pull/5030 first